### DISCUSSION TITLE
Husks and Strays Spawn Underground Tweak [Suggestion]

### DISCUSSION THREAD

**Discussion by Turkey9002** [ACGaming/UniversalTweaks#22] [Tweak Suggestions]
[Charm](https://github.com/svenhjol/Charm/blob/master/src/main/java/svenhjol/charm/mixin/strays_spawn_underground/AllowSpawningAnywhereMixin.java) did this change in newer versions, it's controlled by getCanSpawnHere() in both mobs. Personally I don't understand why Mojang restricted them from spawning underground lol but it's a very simple tweak.

**Comment by ACGaming** [comment]
Yay for simple tweaks! Implemented in https://github.com/ACGaming/UniversalTweaks/commit/3fd3fb5d54ee864fb003c6de5cecbe553827f35f.

**Reply by Turkey9002** [reply]
Awesome! You might want to fix the config description though, it says surface instead of underground. x.x

**Reply by ACGaming** [reply]
Right! That‘s what happens when your brain rephrases mid-sentence. 😁

**Comment by kevin1986xu** [comment]
这是来自QQ邮箱的自动回复邮件。
 
您好，您的邮件已收到，我会尽快给您回复。
